lib/DiscoveryStreamFeed.jsm

```diff
@@ -286,18 +319,22 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
    * @param {RefreshAllOptions} options
    */
   async refreshAll(options = {}) {
-    const dispatch = options.updateOpenTabs ?
+    const {updateOpenTabs, isStartup} = options;
+    const dispatch = updateOpenTabs ?
       action => this.store.dispatch(ac.BroadcastToContent(action)) :
       this.store.dispatch;
 
-    await this.loadLayout(dispatch);
+    const layout = await this.loadLayout(dispatch, isStartup);
     await this.loadComponentFeeds(dispatch);
     await this.loadSpocs(dispatch);
```

Comment:
I would confirm with Nick, but I think layout, feeds, and spocs (so the above three functions) all have caches that need to rely on a week long cache for startups.